File: Daily Test/Java/Expand Each Integer - Matrix 4.10.2025.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int R = sc.nextInt(), C = sc.nextInt();
        String[][] arr = new String[R][C];
        for (int i = 0; i < R; i++)
            for (int j = 0; j < C; j++)
                arr[i][j] = sc.next();
        for (int i = 0; i < R; i++) {
            ArrayList<String> res = new ArrayList<>();
            ArrayList<String> cnt = new ArrayList<>();
            for (int j = 0; j < C; j++) {
                int idx = 0;
                for (char k : arr[i][j].toCharArray()) {
                    if (idx < 2) {
                        res.add(Character.toString(k));
                        idx++;
                    } else {
                        cnt.add(Character.toString(k));
                    }
                }
            }
            System.out.println(String.join(" ", res));
            System.out.println(String.join(" ", cnt));
        }
    }
}

File: Daily Test/Python/Expand Each Integer - Matrix 4.10.2025.py
R, C = map(int, input().split())
arr = [input().split() for _ in range(R)]
for i in range(R):
    res = []
    cnt = []
    for j in range(C):
        idx = 0
        for k in arr[i][j]:
            if idx < 2:
                res.append(k)
                idx += 1
            else:
                cnt.append(k)
    print(' '.join(res))
    print(' '.join(cnt))